README.md

```diff
@@ -1 +1,8 @@
-# dcf-dataservice
\ No newline at end of file
+# dcf-dataservice
+Copy data buckets from GDC to AWS and Google buckets. The jobs need to be run inside a setup kubeneste cluster.
+
+Following the below steps to setup
+- Create a directory in `$vpc_name/apis_configs` and named as `dcf_dataservice`.
+- Put the `manifest` file, `creds.json`, `aws_creds_secret` and `gcloud-creds-secret` into `dcf_dataservice` folder. While `aws_creds_secret` contains AWS key `gcloud-creds-secret` contains google cloud service account.
+- Run `jobs/kube-script-setup.sh`.
```

Comment:
Add a section explaining how the AWS sync works (just uses AWS API to sync in-AWS buckets),
and another explaining how the Google sync works (deploys a data-flow job that copies data from GDC to GCP buckets)

Reply:
+1 diagrams would be nice too so we can easily show this to other parties without having to require reading a block of text